shard.yml

```diff
@@ -1,13 +1,12 @@
 name: invidious
-version: 0.20.1
+version: 2.20240825.2-dev
```

Comment:
How should we handle the `-dev`, in term of the release process? Do we remove it for the release commit, then add a new commit right after that that adds back `-dev`, like so?
```
ac4596fb (HEAD) Prepare for next version
def09876 (v2.2024MMDD.0) Release v2.2024MMDD.0
b1f8ee56 Merge pull request #9876
82f1f891 Fix stuff
```

Reply:
Yeah that's pretty much the only option.

In the Crystal compiler for example, they also just remove and readd the -dev prefix before and after each release https://github.com/crystal-lang/crystal/commits/master/shard.yml